client/public/static/locales/pt/common.json

```diff
@@ -1,74 +1,74 @@
-  "CONFIRM_RESTART_BUILD": "__STRING_NOT_TRANSLATED__",
-  "CONTRIBUTE_GITHUB": "__STRING_NOT_TRANSLATED__",
-  "COPY_BUILD": "__STRING_NOT_TRANSLATED__",
-  "COPY_BUILD_SUCCESS": "__STRING_NOT_TRANSLATED__",
-  "COPY_LINK": "__STRING_NOT_TRANSLATED__",
-  "COPY_LINK_SUCCESS": "__STRING_NOT_TRANSLATED__",
-  "CREATED": "__STRING_NOT_TRANSLATED__",
-  "CREATE_NEW_BUILD": "__STRING_NOT_TRANSLATED__",
-  "DELETE": "__STRING_NOT_TRANSLATED__",
-  "DELETE_BUILD": "__STRING_NOT_TRANSLATED__",
-  "DELETE_BUILD_SUCCESS": "__STRING_NOT_TRANSLATED__",
-  "DUPLICATE_DOFUS_OR_TROPHY": "__STRING_NOT_TRANSLATED__",
-  "DUPLICATE_ITEM_IN_SET": "__STRING_NOT_TRANSLATED__",
-  "EQUIPPED": "__STRING_NOT_TRANSLATED__",
-  "EQUIP_ITEMS": "__STRING_NOT_TRANSLATED__",
-  "EQUIP_ITEMS_plural": "__STRING_NOT_TRANSLATED__",
-  "EQUIP_SET": "__STRING_NOT_TRANSLATED__",
-  "ERROR": "__STRING_NOT_TRANSLATED__",
-  "ERROR_OCCURRED": "__STRING_NOT_TRANSLATED__",
-  "INVALID_CHARACTERISTIC_POINTS": "__STRING_NOT_TRANSLATED__",
-  "INVALID_SCROLLED_CHARACTERISTICS": "__STRING_NOT_TRANSLATED__",
-  "ITEMS": "__STRING_NOT_TRANSLATED__",
-  "JOIN_US_DISCORD": "__STRING_NOT_TRANSLATED__",
-  "LANGUAGE": "__STRING_NOT_TRANSLATED__",
-  "LAST_MODIFIED": "__STRING_NOT_TRANSLATED__",
-  "LEVEL": "__STRING_NOT_TRANSLATED__",
-  "LEVEL_ABBREVIATION": "__STRING_NOT_TRANSLATED__",
-  "MULTIPLE_AP_EXO": "__STRING_NOT_TRANSLATED__",
-  "MULTIPLE_MP_EXO": "__STRING_NOT_TRANSLATED__",
-  "MULTIPLE_PRYSMARADITES": "__STRING_NOT_TRANSLATED__",
-  "MULTIPLE_RANGE_EXO": "__STRING_NOT_TRANSLATED__",
-  "MY_BUILDS": "__STRING_NOT_TRANSLATED__",
-  "NEW_BUILD": "__STRING_NOT_TRANSLATED__",
-  "NO_BONUS": "__STRING_NOT_TRANSLATED__",
-  "NO_BUILDS_MATCHED": "__STRING_NOT_TRANSLATED__",
-  "NO_ITEMS_EQUIPPED": "__STRING_NOT_TRANSLATED__",
-  "NO_PERMISSION": "__STRING_NOT_TRANSLATED__",
-  "NUM_ERROR": "__STRING_NOT_TRANSLATED__",
-  "NUM_ERROR_plural": "__STRING_NOT_TRANSLATED__",
-  "NUM_ITEMS": "__STRING_NOT_TRANSLATED__",
-  "OK": "__STRING_NOT_TRANSLATED__",
-  "OR": "__STRING_NOT_TRANSLATED__",
-  "OWNER": "__STRING_NOT_TRANSLATED__",
-  "REPLACE": "__STRING_NOT_TRANSLATED__",
-  "RESET_ALL": "__STRING_NOT_TRANSLATED__",
-  "RESET_ALL_FILTERS": "__STRING_NOT_TRANSLATED__",
-  "RESTART_BUILD": "__STRING_NOT_TRANSLATED__",
-  "SCROLLED": "__STRING_NOT_TRANSLATED__",
-  "SEARCH": "__STRING_NOT_TRANSLATED__",
-  "SELECT_CLASS": "__STRING_NOT_TRANSLATED__",
-  "SELECT_CLASS_DETAILED": "__STRING_NOT_TRANSLATED__",
-  "SELECT_ITEM_TO_REPLACE": "__STRING_NOT_TRANSLATED__",
-  "SETS": "__STRING_NOT_TRANSLATED__",
-  "STATS_PLACEHOLDER": "__STRING_NOT_TRANSLATED__",
-  "SUCCESS": "__STRING_NOT_TRANSLATED__",
-  "UNTITLED": "__STRING_NOT_TRANSLATED__",
-  "VIEW_ITEMS": "__STRING_NOT_TRANSLATED__",
-  "VIEW_SETS": "__STRING_NOT_TRANSLATED__",
-  "WEAPON_AND_SPELLS": "__STRING_NOT_TRANSLATED__"
+  "ACTIONS": "Ações",
+  "ANONYMOUS": "Anônimo",
+  "AVAILABLE_AT_LEVEL": "Disponível no nível {{level}}",
```

Comment:
I think I have not translated "level". Again, not wrong. But we have to decide what pattern we are going to use.